packages/slate-renderer/src/elements/Embed/lib/injectOembedMarkup.ts

```diff
@@ -13,12 +13,20 @@ const injectOembedMarkup = ({ oembed, onError, target }: Parameters): void => {
     container.innerHTML = oembed.html || '';
     const embedScripts = Array.from(container.getElementsByTagName('script'));
 
+    const currentScriptSources = Array.from(document.getElementsByTagName('script')).map((script) =>
+        script.getAttribute('src'),
+    );
+
     embedScripts.forEach((embedScript) => {
-        const script = document.createElement('script');
-        copyScriptAttributes(embedScript, script);
-        script.addEventListener('error', onError);
+        // Prevent same scripts from loading multiple times
+        if (!currentScriptSources.includes(embedScript.getAttribute('src'))) {
```

Comment:
:x: 

So I have to admit to a terrible overlook from my side... I realized it only after reading [your comment in linear](https://linear.app/prezly/issue/DEV-2184#comment-44026fc9).

Please create an html file with the following content and open it in your browser. Then open up developer tools, refresh the page and answer this question: how many times has `https://cdn.jsdelivr.net/lodash/4/lodash.min.js` been loaded?

```html
<!DOCTYPE html>
<html>
<head>
  <script src="https://cdn.jsdelivr.net/lodash/4/lodash.min.js"></script>
  <script src="https://cdn.jsdelivr.net/lodash/4/lodash.min.js"></script>
</head>
<body>
  <script src="https://cdn.jsdelivr.net/lodash/4/lodash.min.js"></script>
  <script src="https://cdn.jsdelivr.net/lodash/4/lodash.min.js"></script>
</body>
</html>
```

Reply:
Yeah, it's loaded only one time 🤔 
So the I guess this PR can be safely discarded :)

Reply:
@kamilmielnik strangely I don't have option to close the pull request, can you do it please?